src/bf_dpu_update.py

```diff
@@ -362,37 +362,46 @@ def update_bfb(self):
             if protocol is None:
                 raise Err_Exception(Err_Num.NOT_SUPPORT_SIMPLE_UPDATE_PROTOCOL, 'The current supported BFB update protocols are {}'.format(protocols))
 
-        return (protocol, self.update_bfb_by_protocol(protocol))
+        return (protocol, self.simple_update_by_protocol(protocol))
 
 
-    def update_bfb_by_protocol(self, protocol):
+    def simple_update_by_protocol(self, protocol):
         if protocol == 'HTTP':
-            return self.update_bfb_by_http()
+            return self.simple_update_by_http()
         elif protocol == 'SCP':
-            return self.update_bfb_by_scp()
+            return self.simple_update_by_scp()
 
 
-    def update_bfb_impl(self, protocol, image_uri):
+    def get_simple_update_targets(self):
+        if self.module == 'BIOS':
+            return ['redfish/v1/UpdateService/FirmwareInventory/DPU_OS']
+        elif self.module == 'CONFIG':
+            return ["redfish/v1/UpdateService/FirmwareInventory/golden_image_config"]
+        else:
+            raise Err_Exception(Err_Num.UNSUPPORTED_MODULE, "Only BIOS and CONFIG can be updated by SimpleUpdate")
+
+
+    def simple_update_impl(self, protocol, image_uri):
         url = self._get_url_base() + '/UpdateService/Actions/UpdateService.SimpleUpdate'
         headers = {
             'Content-Type'     : 'application/json'
         }
         data = {
             'TransferProtocol' : protocol,
             'ImageURI'         : image_uri,
-            'Targets'          : ['redfish/v1/UpdateService/FirmwareInventory/DPU_OS'],
+            'Targets'          : self.get_simple_update_targets(),
             'Username'         : self._get_local_user()
         }
         response = self._http_post(url, data=json.dumps(data), headers=headers)
-        self.log('Update BFB Firmware', response)
+        self.log('Simple Update', response)
```

Comment:
simple update to 'Update BFB or Configurations'?

Reply:
Will change to "Do Simple Update (Update BFB or Configurations ... ...)"